packages/graphql/src/mutations/_txHelper.js

```diff
@@ -426,8 +428,8 @@ async function sendViaRelayer({
     to
   })
 
-  if (!resp) {
-    throw new Error('No response from relayer!')
+  if (!resp || !resp.id) {
+    throw new Error('No transaction hashfrom relayer!')
```

Comment:
missing space.

Reply:
I'm terrible at English.

Reply:
I'm OCD with spelling. Perhaps I was traumatized at an early age by my mom who is a teacher... !